src/main.js

```diff
@@ -15,7 +16,7 @@ const store = createStore(initialState)
 const MOUNT_NODE = document.getElementById('root')
 
 let render = () => {
-  const routes = require('./routes/index').default(store)
```

Comment:
Using require this way waits until it is called to load the dependency. Using imports imports regardless.